src/main/java/hudson/os/windows/ManagedWindowsServiceLauncher.java

```diff
@@ -100,18 +100,6 @@
      */
     private ManagedWindowsServiceAccount account;
     
-    public static class LogOnOption {
```

Comment:
I am not inclined to keep a member that was public only for databinding purposes and is not referenced from other @jenkinsci code.